modules/eco-self-node-remediation-operator-about.adoc

```diff
@@ -55,10 +55,16 @@ If a watchdog device is unavailable, the `SelfNodeRemediationConfig` CR uses a s
 [id="understanding-self-node-remediation-remediation-template-config_{context}"]
 == Understanding the Self Node Remediation Template configuration
 
-The Self Node Remediation Operator also creates the `SelfNodeRemediationTemplate` CR with the name `self-node-remediation-default-template` in the Self Node Remediation Operator's namespace. This CR defines the remediation strategy for the nodes.
+The Self Node Remediation Operator also creates the `SelfNodeRemediationTemplate` Custom Resource Definition (CRD). This CRD defines the remediation strategy for the nodes. The following remediation strategies are available:
 
-The default remediation strategy is `NodeDeletion` that removes the `node` object.
-In {product-title} 4.10, the Self Node Remediation Operator introduced a new remediation strategy called `ResourceDeletion`. The `ResourceDeletion` remediation strategy removes the pods and associated volume attachments on the node rather than the `node` object. This strategy helps to recover workloads faster.
+ResourceDeletion:: This remediation strategy removes the pods and associated volume attachments on the node rather than the node object. This strategy helps to recover workloads faster. `ResourceDeletion` is the default remediation strategy.
+
+NodeDeletion:: This remediation strategy removes the node object.
+
+The Self Node Remediation Operator creates the following `SelfNodeRemediationTemplate` CRs for each strategy:
+
+* `self-node-remediation-resource-deletion-template`, which the `ResourceDeletion` remediation strategy uses.
+* `self-node-remediation-node-deletion-template`, which the `NodeDeletion` remediation strategy uses.
 
 The `SelfNodeRemediationTemplate` CR resembles the following YAML file:
```

Comment:
This might be out of context for this PR, but I'll mention it for reference. You could replace "resembles" with something more definite. As it stands, the reader might interpret this to mean that the YAML for the CR is somewhat like this example.

Reply:
Makes sense. Noted.I will create a task for it. Thanks!

Reply:
Thank you!